boilerplate/w3c/copyright.include

```diff
@@ -1 +1 @@
-<a href="https://www.w3.org/Consortium/Legal/ipr-notice#Copyright">Copyright</a> © [YEAR] <a href="https://www.w3.org/"><abbr title="World Wide Web Consortium">W3C</abbr></a><sup>®</sup> (<a href="https://www.csail.mit.edu/"><abbr title="Massachusetts Institute of Technology">MIT</abbr></a>, <a href="https://www.ercim.eu/"><abbr title="European Research Consortium for Informatics and Mathematics">ERCIM</abbr></a>, <a href="https://www.keio.ac.jp/">Keio</a>, <a href="https://ev.buaa.edu.cn/">Beihang</a>). W3C <a href="https://www.w3.org/Consortium/Legal/ipr-notice#Legal_Disclaimer">liability</a>, <a href="https://www.w3.org/Consortium/Legal/ipr-notice#W3C_Trademarks">trademark</a> and <a rel="license" href="https://www.w3.org/Consortium/Legal/2015/copyright-software-and-document">permissive document license</a> rules apply.
+<a href="https://www.w3.org/Consortium/Legal/ipr-notice#Copyright">Copyright</a> © [YEAR] <a href="https://www.w3.org/">World Wide Web Consortium, Inc.</a>. <abbr title="World Wide Web Consortium">W3C</abbr>®</sup> <a href="https://www.w3.org/Consortium/Legal/ipr-notice#Legal_Disclaimer">liability</a>, <a href="https://www.w3.org/Consortium/Legal/ipr-notice#W3C_Trademarks">trademark</a> and <a rel="license" href="https://www.w3.org/Consortium/Legal/2015/copyright-software-and-document">permissive document license</a> rules apply.
```

Comment:
That lone `</sup>` looks bogus.

Also, are you sure you want that comma there? It's not in the title of this PR...

Reply:
I meant the comma before Inc., to be clear. (The superscript also seems somewhat redundant to me given the character used, but I guess that is how it is done already.)

Reply:
so far, the information I have tells me it's "World Wide Web Consortium, Inc" but I'm double checking to be sure.

On the <sup>, it came from the previous markup. Looking around, it does seem indeed that the (R) symbol isn't used with superscript, unlike (tm) or (sm).

Reply:
So, it's really "[World Wide Web Consortium, Inc.](https://opencorporates.com/companies/us_de/7614104)".

Reply:
and for ®, the float above the baseline varies depending on the font used. So I guess we should stick with <sup> for now.